# -*- coding:utf-8 -*-

from saklient.errors.httpserviceunavailableexception import HttpServiceUnavailableException
from saklient.errors.httpserviceunavailableexception import HttpServiceUnavailableException

# module saklient.cloud.errors.storageoperationfailureexception

class StorageOperationFailureException(HttpServiceUnavailableException):
    ## サービスが利用できません。ストレージの操作に失敗しました。サーバが混雑している可能性があります。
    
    # (class field) default_message = "サービスが利用できません。ストレージの操作に失敗しました。サーバが混雑している可能性があります。"
    
    pass
